qa/workunits/rbd/rbd_mirror_snapshot.sh

```diff
@@ -445,6 +486,12 @@ demote_image ${CLUSTER1} ${POOL} ${image}
 wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image} 'up+error' 'split-brain'
 request_resync_image ${CLUSTER1} ${POOL} ${image} image_id
 wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image} 'up+replaying'
+remove_image_retry ${CLUSTER2} ${POOL} ${image}
+
+testlog "TEST: check if removed images' OMAP are cleanup at startup"
```

Comment:
This test is failing and I can see this in the cleanup status:
```
            "image_replayers": [
                {
                    "name": "mirror/0e8ace18-4d09-4810-8203-d8451cae8dc4",
                    "state": "Stopped"
                }
            ]
```
(note that the qa patch is still not great, I am working on it